### DISCUSSION TITLE
Discord-AI-Selfbot for PC has not announced today, tomorrow for sure.

### DISCUSSION THREAD

**Discussion by pnydiffuser** [Najmul190/Discord-AI-Selfbot#51] [General]
![Untitled](https://github.com/user-attachments/assets/2d69190e-6710-4c24-a21c-bbcc9a57d7cd)

**Comment by Najmul190** [comment]
![giphy-ezgif com-webp-to-gif-converter](https://github.com/user-attachments/assets/6a6bd352-eba2-4e2a-aaff-a8a3d30e0523)

**Reply by pnydiffuser** [reply]
YESS! I never heard of Groq, but it's been the perfect platform for super quick text (faster than gpt/gemini token output!?)
Would be willing to commision Gemini 1.5 Flash/Pro models into the mix, to hopefully improve -analyze accuracy, and for it's huge context window.

**Reply by Najmul190** [reply]
I'd love to add Gemini, but for some reason there's no free tier in the UK so I have no idea how to implement it 🥲

![image](https://github.com/user-attachments/assets/b6e566aa-136d-46db-8ea7-d9396806ff7d)

**Reply by pnydiffuser** [reply]
> I'd love to add Gemini, but for some reason there's no free tier in the UK so I have no idea how to implement it 🥲
> 
> ![image](https://private-user-images.githubusercontent.com/68229089/354228632-b6e566aa-136d-46db-8ea7-d9396806ff7d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI2NjM2NzgsIm5iZiI6MTcyMjY2MzM3OCwicGF0aCI6Ii82ODIyOTA4OS8zNTQyMjg2MzItYjZlNTY2YWEtMTM2ZC00NmRiLThlYTctZDkzOTY4MDZmZjdkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAzVDA1MzYxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI0YTFjM2EyZDc3NDVlMGFmMzBiNWJjNzI5YjljOGQ0OTQ0NmVlNDkyMjA0MDI3Y2UyZTc4ZTY3ZWVlOTMxYjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DI-Bi0FP9P0n-Ne6JtEjOn1_X5fDuOyslT4VdikpO0s)

Ohh! Only way i know to get around this is ProtonVPN to make a temp google at USA free vpn servers, add in Gemini, and let whoever can insert their own API key.
